.yamllint

```diff
@@ -5,6 +5,8 @@ rules:
   line-length:
     max: 120
     level: warning
+  truthy:
+    allowed-values: ['true', 'false', 'yes', 'no']
```

Comment:
Alternatively, I don't really like allowing yes/no just to be explicit it's a bool, so can you leave this out and change the value to `true` in the task? That's the style I've adopted in all my roles and Ansible projects.

Reply:
Ahhh right, I misunderstood your first comment. Sure, I will prepare a fix for it at some point this weekend.